uwblueprint/paramedics-react
# Resource management cleanup

[Notion ticket](https://www.notion.so/uwblueprintexecs/resource-management-cleanup-03dbb3984c934be5abaaba2170d517ab) 
[Notion ticket 2](https://www.notion.so/uwblueprintexecs/fix-back-button-in-resource-mgmt-a727d5e4e0bb4e3fb3a4d6ff3928e5fb)

## Changes
- Adds functionality to back button, right-aligns refresh button on resource management page
- adds sticky behaviour to resource management header and tabs
- moves enqueue snackbar to onCompleted callback for resource mutation hooks
- adds validation for unique user emails on add member page (checks for duplicate emails)
- disables editing of user emails 


## Screenshots

## Testing
- Navigate to Resource Management Page
  - Ensure back button works and refresh button is properly aligned
  - Scroll on the page, ensure tabs and header "stick" to top of page
- Add a user (aka team member) 
  - Ensure page doesn't allow adding a user with a duplicate email
- Edit a user
  - Ensure email can't be edited
- Ensure snackbar notifications appear after adding/editing a user

## Checklist
>[Code review doc for reference](https://www.notion.so/uwblueprintexecs/Code-Review-a21fc85b00394f488e92d9d605f6b2bc)

before opening PR

filling out PR

after opening PR
